src/accordion/accordion.ts

```diff
@@ -45,27 +17,48 @@ export class NgbAccordion {
     </div>
   `
 })
-export class NgbAccordionGroup implements OnDestroy {
-  private isDisabled: boolean;
-  private _isOpen: boolean = false;
+export class NgbAccordionGroup {
+  private _isOpen = false;
+  private accordion: NgbAccordion;
+  @Input() isDisabled: boolean;
+  @Input() heading: string;
+
+  @Input()
+  set isOpen(value: boolean) {
+    this._isOpen = value;
+    if (value) {
+      this.accordion.closeOthers(this);
+    }
+  }
 
-  constructor(private accordion: NgbAccordion) { this.accordion.addGroup(this); }
+  get isOpen(): boolean { return this._isOpen; }
 
-  toggleOpen(event) {
+  constructor(@Inject(forwardRef(() => NgbAccordion)) accordion: NgbAccordion) { this.accordion = accordion; }
+
+  toggleOpen(event): void {
     event.preventDefault();
     if (!this.isDisabled) {
       this.isOpen = !this.isOpen;
     }
   }
+}
 
-  onDestroy(): void { this.accordion.removeGroup(this); }
+@Directive({selector: 'ngb-accordion, [ngb-accordion]'})
```

Comment:
Shouldn't it be a component with its own template? AFAIK we still need some markup, from the bs4 docs: `<div id="accordion" role="tablist" aria-multiselectable="true">`